pkg/remote/trans_errors.go

```diff
@@ -36,6 +36,8 @@ const (
 	InvalidTransform            = 8
 	InvalidProtocol             = 9
 	UnsupportedClientType       = 10
+	// kitex's own type id from number 20
+	UnknownService = 20
```

Comment:
Why don't use 11?

Reply:
Because these ids correspond to the definition in thrift TApplicationException. So, it's safer to skip some numbers in case TApplicationException adds new ids.